### PR TITLE
feat: CommonsChunkPlugin indexOf -> includes

### PR DESCRIPTION
**Pull request by Iwouldliketobeapig**

### CODE REVIEW

File: lib/optimize/CommonsChunkPlugin.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ Take a look at the "name"/"names" or async/children option.`);
 		if(Array.isArray(selectedChunks)) {
 			return allChunks.filter(chunk => {
 				const notCommmonChunk = chunk !== targetChunk;
-				const isSelectedChunk = selectedChunks.indexOf(chunk.name) > -1;
+				const isSelectedChunk = selectedChunks.includes(chunk.name)
```
Comment (Kovensky) [inline lib/optimize/CommonsChunkPlugin.js]: `;`

**Comment by TheLarkInn** [comment]
Have you signed the CLA?! If so I believe this is approvable.

**Comment by Iwouldliketobeapig** [comment]
what should i do, this link is bad to cla.（ I am trying to read webpack code ）

**Comment by Iwouldliketobeapig** [comment]
I just want to know my thoughts right or not. so i am do it .This is my first time, it's nervous.

**Comment by sokra** [comment]
Thanks

**Pull request merged**